pkgs/by-name/uv/uv/package.nix

```diff
@@ -1,19 +1,18 @@
 {
+  stdenv,
   lib,
   cmake,
   fetchFromGitHub,
   installShellFiles,
   pkg-config,
-  python3Packages,
   rustPlatform,
   versionCheckHook,
   nix-update-script,
 }:
 
-python3Packages.buildPythonApplication rec {
+stdenv.mkDerivation rec {
```

Comment:
Sorry, but if this package can be built using the standard `rustPlatform.buildRustPackage`, then please, use it.